docs/maintainers/docker-image.rst

```diff
@@ -0,0 +1,50 @@
+Building Docker Images
+======================
```

Comment:
> note: this documentation is for ListenBrainz maintainers for when they deploy the website

or something like this?

Reply:
This is listed under the maintainers section so don't think its needed. But then we don't have maintainers documentation index page so probably fine to add.

Reply:
right, I was going to suggest to put this on the main "maintainers page", but it seems that we don't have one of them? which is why I suggested here

Reply:
Yeah, makes sense. Another PR adds a few more pages to maintainer docs but no index page there as welll. I'll add one there and then we can the note from here.

Reply:
perfect